README.md

```diff
@@ -4,12 +4,25 @@
 This is app a Play app for editing media atoms, and a
 publisher that pushes media atoms into a Kinesis stream.
 
+## Credentials
+0. You will need `media-service` credentials to run the app and `composer`
+   credentials if you want to be able to publish media atoms.
+0. You will need read access to S3 files in the `workflow` profile for panda
+   auth to work.
+0. You must also have a region set in `~/.aws/config`:
```

Comment:
`aws configure set region eu-west-1 --profile media-service` might be more resilient if aws changes their config structure